src/TabView.tsx

```diff
@@ -49,6 +57,7 @@ export default function TabView<T extends Route>({
   style,
   swipeEnabled = true,
   tabBarPosition = 'top',
+  tabBarSpacing = 0,
```

Comment:
The prop should directly be passed to the TabBar, doesn't make sense to pass it to TabView when all it would do is pass it to TabBar